### PR TITLE
starts on automation for creation

### PR DESCRIPTION
**Pull request by jread-usgs**
partial for #90

### CODE REVIEW

**Comment by jread-usgs** [comment]
this renders to create this:
```XML
<?xml version="1.0"?>
<metadata>
  <idinfo/>
  <citation>
    <citeinfo>
      <origin>Alison P Appling</origin>
      <origin>Jordan S Read</origin>
      <origin>a bunch of other peeps too!</origin>
      <pubdate>2017</pubdate>
      <title>Metabolism data for awesome things!</title>
      <geoform>text files</geoform>
      <onlink>http://dx.doi.org/10.5066/THIS.IS.FAKE</onlink>
    </citeinfo>
  </citation>
  <descript>
    <abstract>FAKE It is well recognized that the climate is warming in response to anthropogenic emission of greenhouse gases. Over the last decade, this has had a warming effect on lakes. Water clarity is also known to effect water temperature in lakes. What is unclear is how a warming climate might interact with changes in water clarity in lakes. As part of a project at the USGS Office of Water Information, several water clarity scenarios were simulated for lakes in Wisconsin to examine how changing water clarity interacts with climate change to affect lake temperatures at a broad scale. This data set contains the following parameters: year, WBIC, durStrat, max_schmidt_stability, mean_schmidt_stability_JAS, mean_schmidt_stability_July, SthermoD_mean_JAS, SthermoD_mean, lake_average_temp, peak_lake_average_temp, lake_average_temp_JAS, mean_epi_temp, mean_hypo_temp, mean_surf_temp, mean_bottom_temp, peak_surf_temp, peak_bottom_temp, mean_surf_temp_JAS, mean_bottom_temp_JAS, mean_bottom_temp_365, mean_surf_temp_365, mean_1m_temp, mean_surf_JA, GDD_wtr_5c, GDD_wtr_10c, volume_mean_m_3, simulation_length_days, mean_volumetric_temp, kd, out_val calculated for 2210 lakes</abstract>
    <purpose>To bring metabolism data to the world!</purpose>
  </descript>
  <timeperd>
    <timeinfo>
      <rngdates>
        <begdate>20070101</begdate>
        <enddate>2016XXXX</enddate>
      </rngdates>
    </timeinfo>
  </timeperd>
  <status>
    <progress>Complete</progress>
    <update>update planned for year 2020</update>
  </status>
  <spdom>
    <bounding>
      <westbc>-109</westbc>
      <eastbc>-68</eastbc>
      <northbc>52</northbc>
      <southbc>26.3</southbc>
    </bounding>
  </spdom>
  <keywords>
    <theme>
    <themekt>none</themekt>
    <themekey>water</themekey>
        <themekey>streams</themekey>
        <themekey>rivers</themekey>
        <themekey>metabolism</themekey>
        <themekey>primary productivity</themekey>
        <themekey>carbon</themekey>
    </theme>
    <theme>
      <themekt>ISO 19115 Topic Category</themekt>
      <themekey>environment</themekey>
      <themekey>inlandWaters</themekey>
      <themekey>007</themekey>
      <themekey>012</themekey>
    </theme>
    <place>
      <placekt>Department of Commerce, 1995, Countries, Dependencies, Areas of Special Sovereignty, and 
                Their Principal Administrative Divisions,  Federal Information Processing Standard (FIPS) 10-4, 
                Washington, D.C., National Institute of Standards and Technology</placekt>
      <placekey>United States</placekey>
      <placekey>US</placekey>
    </place>
    <place>
			<placekt>U.S. Department of Commerce, 1987, Codes for the identification of the States, the District of Columbia and the outlying areas of the United States, and associated areas (Federal Information Processing Standard 5-2): Washington, D. C., NIST</placekt>
      <placekey>Wisconsin</placekey>
      <placekey>WI</placekey>
    </place>
    <place>
			<placekt>U.S. Department of Commerce, 1987, Codes for the identification of the States, the District of Columbia and the outlying areas of the United States, and associated areas (Federal Information Processing Standard 5-2): Washington, D. C., NIST</placekt>
      <placekey>New Hampshire</placekey>
      <placekey>NH</placekey>
    </place>
    <place>
      <placekt>none</placekt>
      <placekey>Wisconsin</placekey>
    </place>
<place>
      <placekt>none</placekt>
      <placekey>New Hampshire</placekey>
    </place>
  </keywords>
  <accconst>none</accconst>
  <useconst>These data are subject to change and are not citable until reviewed and approved for official publication by the USGS</useconst>
  <ptcontac>
    <cntinfo>
      <cntperp>
        <cntper></cntper>
        <cntorg>U.S. Geological Survey</cntorg>
      </cntperp>
    </cntinfo>
    <cntaddr>
      <addrtype>Mailing and Physical</addrtype>
      <address>8551 Research Way</address>
      <city>Middleton</city>
      <state>WI</state>
      <postal>53562</postal>
      <country>U.S.A.</country>
    </cntaddr>
    <cntvoice></cntvoice>
    <cntemail></cntemail>
  </ptcontac>
  <datacred></datacred>
  <native></native>
  <crossref>
    <citeinfo>
      <origin></origin>
      <pubdate></pubdate>
      <title></title>
      <geoform></geoform>
      <pubinfo>
        <pubplace></pubplace>
        <publish></publish>
      </pubinfo>
    </citeinfo>
  </crossref>
  <dataqual>
    <attracc>
      <attraccr>No formal attribute accuracy tests were conducted.</attraccr>
    </attracc>
    <logic>not applicable</logic>
    <complete>not applicable</complete>
  </dataqual>
  <posacc>
    <horizpa>
      <horizpar>A formal accuracy assessment of the horizontal positional information in the data set has not been conducted.</horizpar>
    </horizpa>
    <vertacc>
      <vertaccr>A formal accuracy assessment of the vertical positional information in the data set has either not been conducted, or is not applicable.</vertaccr>
    </vertacc>
  </posacc>
  <lineage>
    <procstep>
      <procdesc></procdesc>
      <procdate></procdate>
    </procstep>
  </lineage>
  <spdoinfo>
    <indspref></indspref>
    <direct>Point</direct>
    <ptvctinf>
      <sdtsterm>
        <sdtstype>Point</sdtstype>
        <ptvctcnt></ptvctcnt>
      </sdtsterm>
    </ptvctinf>
  </spdoinfo>
  <spref>
    <horizsys>
      <geograph>
        <latres></latres>
        <longres></longres>
        <geogunit>Decimal degrees</geogunit>
      </geograph>
      <geodetic>
        <horizdn>North American Datum of 1983</horizdn>
        <ellips>Geodetic Reference System 80</ellips>
        <semiaxis>6378137.0</semiaxis>
        <denflat>298.257</denflat>
      </geodetic>
    </horizsys>
  </spref>
  <eainfo>
    <detailed>
      <enttyp>
        <enttypl></enttypl>
        <enttypd></enttypd>
        <enttypds>Producer Defined</enttypds>
      </enttyp>

    </detailed>
  </eainfo>
  <distinfo>
    <distrib>
      <cntinfo>
        <cntperp>
          <cntper></cntper>
          <cntorg>U.S. Geological Survey - ScienceBase</cntorg>
        </cntperp>
        <cntaddr>
          <addrtype>Mailing and Physical</addrtype>
          <address>Denver Federal Center, Building 810, Mail Stop 302</address>
          <city>Denver</city>
          <state>CO</state>
          <postal>80255</postal>
          <country>U.S.A.</country>
        </cntaddr>
        <cntvoice>1-888-275-8747</cntvoice>
        <cntemail>sciencebase@usgs.gov</cntemail>
      </cntinfo>
    </distrib>
    <distliab></distliab>
    <stdorder>
      <digform>
        <digtinfo>
          <formname></formname>
          <formvern>none</formvern>
        </digtinfo>
        <digtopt>
          <onlinopt>
            <computer>
              <networka>
                <networkr>http://dx.doi.org/10.5066/THIS.IS.FAKE</networkr>
              </networka>
            </computer>
          </onlinopt>
        </digtopt>
      </digform>
      <fees>None</fees>
    </stdorder>
  </distinfo>
  <metainfo>
    <metd></metd>
    <metc/>
    <cntinfo>
      <cntperp>
        <cntper></cntper>
        <cntorg>U.S. Geological Survey</cntorg>
      </cntperp>
      <cntpos>Data Chief</cntpos>
      <cntaddr>
        <addrtype>Mailing and Physical</addrtype>
        <address>8551 Research Way #120</address>
        <city>Middleton</city>
        <state>WI</state>
        <postal>53562</postal>
        <country>U.S.A.</country>
      </cntaddr>
      <cntvoice></cntvoice>
      <cntfax>608 821-3817</cntfax>
      <cntemail></cntemail>
    </cntinfo>
    <metstdn>FGDC Biological Data Profile of the Content Standard for Digital Geospatial Metadata</metstdn>
    <metstdv>FGDC-STD-001.1-1999</metstdv>
  </metainfo>
</metadata>
```
which is missing some fields (those that aren't in the yaml)

**Comment by aappling-usgs** [comment]
wanna add `respiration` to the `themekey`s?

**Comment by jread-usgs** [comment]
oh definitely (and I will add that), but I am filling in mostly placeholder text so that a co-author can go in later and fill in the real stuff **and** I can figure out what should be programmatically extracted (like bounding box and states) and what should be manual.

**Comment by aappling-usgs** [comment]
sounds good

**Comment by aappling-usgs** [comment]
i'm still hoping for navigation comments in metadata_create_timeseries.R

**Comment by jread-usgs** [comment]
what are you referring to? navigation comments?

**Comment by aappling-usgs** [comment]
my comment here: https://github.com/USGS-CIDA/stream_metab_usa/pull/93/files#r68399573

> what do you think about adding some comment headers at several points within this document, e.g., to distinguish contact information from dataset location information? a few of those could help with navigating the document and getting a quick high-level sense of what sort of information is included here.

**Comment by aappling-usgs** [comment]
:100:

File: data_release/metadata_create_timeseries.R [hunk 1]
```diff
@@ -63,7 +72,9 @@ k %>%
 
 k %>% 
   xml_add_child('state-template')
-  
+# </---Keywords--->
```
Comment (jread-usgs) [inline data_release/metadata_create_timeseries.R]: added roadmap @aappling-usgs. I think this is what you were talking about

**Pull request merged**